### PR TITLE
Update example.md

### PR DESCRIPTION
**Pull request by csalam07**
Added installation instruction at the top section. This will be helpful for the beginners or freshers to get started with next-auth.

## ☕️ Reasoning

What changes are being made? What feature/bug is being fixed here?

## 🧢 Checklist

- [x] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 🎫 Affected issues

Please [scout and link issues](https://github.com/nextauthjs/next-auth/issues) that might be solved by this PR.

Fixes: INSERT_ISSUE_LINK_HERE

## 📌 Resources

- [Contributing guidelines](./CONTRIBUTING.md)
- [Code of conduct](./CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

**Comment by csalam07** [comment]
I wasted several minutes searching the installation guide. Later I had to refer it from [www.npmjs.com](url). It would be better if the installation guide is located at eyesight. So I added it. I believe this will be helpful for any fresher or beginner at Next-Auth.

File: docs/docs/getting-started/example.md [hunk 1]
```diff
@@ -3,6 +3,21 @@ id: example
 title: Getting Started
 ---
 
+## Installtion
```
Comment (ndom91) [inline docs/docs/getting-started/example.md:6]:
```suggestion
## Installation
```

File: docs/docs/getting-started/example.md [hunk 2]
```diff
@@ -3,6 +3,21 @@ id: example
...
+
+```sh
+  npm i next-auth
+```
+
+Install with yarn
+
+```sh
+  yarn add next-auth
+```
```
Comment (ndom91) [inline docs/docs/getting-started/example.md:18]: Also we have a plugin which will automatically make an `npm`/`yarn` tabbed instruction thing.

For example:

```
You can upgrade to the new version by running:

` ` ` bash npm2yarn
npm install next-auth
` ` `
```

Will generate:

image
image


See `/docs/getting-started/upgrade-to-v4.md:16`

**Review by ndom91 — COMMENTED** [review]
Few notes, otherwise, not a bad idea.

**Comment by balazsorban44** [comment]
Closing as stale

**Pull request closed without merging**